os/arch/arm/include/armv7-a/irq.h

```diff
@@ -381,15 +379,15 @@ static inline irqstate_t irqsave(void)
       : "memory"
     );
 
-  return cpsr;
+	return cpsr;
 }
 
 /* Enable IRQs and return the previous IRQ state */
 
 static inline irqstate_t up_irq_enable(void)
 {
-  unsigned int cpsr;
-
+	unsigned int cpsr;
+	
   __asm__ __volatile__
```

Comment:
ASM code is not modified.... Please find my comment before I left.

Reply:
Ok, I will modify it.